README.md

```diff
@@ -32,68 +79,144 @@ and [test](https://en.wikipedia.org/wiki/Training,_validation,_and_test_sets#Tes
 This terminology has lead to [some confusion in the past](https://en.wikipedia.org/wiki/Training,_validation,_and_test_sets#Confusion_in_terminology). 
 Splitting the `hindcast-input` (`training-input`) dataset between training and validation is standard way and should be decided carefully.
 
-The `forecast-input` (`test-input`) must not be used as a validation dataset  : it must not be used to tune the hyperparameters or make decision about the ML model. 
-Fostering discussions about how to prevent overfitting may be an outcome of the challenge.
+The `forecast-input` (`test-input`) must not be used as a validation dataset: it must not be used to tune the hyperparameters or make decision about the ML model. 
 
-### Hindcast input (Training input)
+
+### Hindcast input
 
 The `hindcast-input`(`training-input`) dataset consists of data from three different models: ECMWF (ecmf), ECCC (cwao), NCEP (kwbc).
 Use `origin="ecmwf"` (model name) or `origin="ecmwf"` (center name).
 These data are hindcast data. This is used as the input for training the ML models.
-This dataset is available as `grib`, `netcdf` or `zarr`.
+This dataset is available as `format`: `grib`, `netcdf` or `zarr`.
 In this dataset, the data is available from 1999 for the oldest, to 2019/12/31 for the most recent. <!-- which model is available from 1998? -->
   - ECMWF hindcast data
-    - `forecast_time`: from 2000/01/02 to 2019/12/31, weekly every 7 days (every Thurday).
+    - `forecast_time`: from 2000/01/02 to 2019/12/31, corresponding to the weekly Thurdays in 2020.
     - `lead_time`: 0 to 46 days
     - `valid_time` (`forecast_time` + `lead_time`): from 2000/01/02 to 2020/02/13
     - availables parameters : `t2m(2t)/siconc(ci)/gh/lsm/msl/q/rsn/sm100/sm20/sp/sst/st100/st20/t/tcc/tcw/tp/ttr/u/v` (differing name in [MARS database](https://confluence.ecmwf.int/display/S2S/Parameters))
   - ECCC hindcast data 
-    - `forecast_time`: from to , weekly every 7 days (every Thurday).
+    - `forecast_time`: from 2000/01/02 to 2019/12/31, corresponding to the weekly Thurdays in 2020.
     - `lead_time`: 1 to 32 days
     - `valid_time` (forecast_time + lead_time): from 
     - availables parameters: `t2m(2t)/siconc(ci)/gh/lsm/msl/q/rsn/sp/sst/t/tcc/tcw/tp/ttr/u/v` (differing name in [MARS database](https://confluence.ecmwf.int/display/S2S/Parameters))
-    - parameters not available: sm20, sm100, st20, st100
+    - parameters not available: `sm20,sm100,st20,st100`
   - NCEP hindcast data 
-    - `forecast_time` : from 1999/01/07 to 2010/12/30, weekly every 7 days (every Thurday).
+    - `forecast_time` : from 1999/01/07 to 2010/12/30, corresponding to the weekly Thurdays in 2010.
```

Comment:
Yes, it is better.
Maybe point that "Notice that the Thursday in 2010 do not happen the same days-of-years as the Thursdays in 2020", therefore the forecast_time dates in the NCEP data will not match the forecast_time in the ECCC and ECMWF data. For instance, 02.01.2009 is in the ECCC and ECMWF data (02.01.2020 is a Thurday) but 02.01.2009 is not in NCEP data (02.01.2010 is not a Thurday)".

Reply:
your explanation is right but I think a bit long to put it here.

Reply:
fair point. As long as it is somewhere, that's fine.